package/network/ipv6/odhcp6c/files/dhcpv6.script

```diff
@@ -237,5 +237,8 @@ esac
 
 # user rules
 [ -f /etc/odhcp6c.user ] && . /etc/odhcp6c.user "$@"
+for f in /etc/odhcp6c.user.d/*; do
+  [ -e "$f" ] && (. "$f" "$@")
```

Comment:
Is there a specific reason why you use -e and not -f ?

Reply:
no other than symbolic links. should i change it?

Reply:
I'm in favor of keeping the checks identical with the existing check two lines above if there's no specific reason to use -f

Reply:
sure :-)

done